### PR TITLE
MaCH-1.0.18.c-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 1 out of 9
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/1243263eef7f5d31146a for a full test report.

File: easybuild/easyconfigs/m/MaCH/MaCH-1.0.18.c-ictce-6.2.5.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MaCH/MaCH-1.0.18.c-ictce-6.2.5.eb]: minor changes needed here, `easyblock = 'ConfigureMake'` , put this in REVIEW plz!

File: easybuild/easyconfigs/m/MaCH/MaCH-1.0.18.c-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/m/MaCH/MaCH-1.0.18.c-goolf-1.4.10.eb]: ditto, on easyblock=

**Comment by pescobar** [comment]
Test report by @pescobar
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux centos 6.5, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/a4e16f896bb3247b30f7 for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6d6fdc6dffc22c8e5cdc for a full test report.

**Comment by boegel** [comment]
missing dep on `zlib`:

```
g++ -O2 -march=native -O2 -static -I./libsrc -I./mach1 -D__ZLIB_AVAILABLE__  -D_FILE_OFFSET_BITS=64 -D_LARGEFILE64_SOURCE -o executables/mach1 mach1/AssociationAnalysis.o mach1/CostCalculator.o mach1/DosageCalculator.o mach1/ErrorRate.o mach1/Manners.o mach1/Haplotyper.o mach1/HaplotypeKey.o mach1/HaplotypeLoader.o mach1/MergeHaplotypes.o mach1/OutputHandlers.o mach1/Main.o libsrc/lib-goncalo.a  -lm -lz
/usr/bin/ld: cannot find -lz
```

**Comment by boegel** [comment]
@pescobar ping on missing dep?

**Comment by pescobar** [comment]
shit I did something stupid with git. closing this one

**Comment by boegel** [comment]
replaced by #3136

**Pull request closed without merging**